src/mmw/js/src/core/views.js

```diff
@@ -59,6 +61,10 @@ var HeaderView = Marionette.ItemView.extend({
         // Defer requiring app until needed as it is not defined when
         // core.views are initialized (they are required in app.js)
         require('../app').showLoginModal();
+    },
+
+    userLogout: function() {
+        require('../app').userLogout();
```

Comment:
Since this gets required more than once in this module, consider moving it with the rest of the require statements and updating these references.